query/language/redefine.feature

```diff
@@ -98,13 +97,20 @@ Feature: TypeQL Redefine Query
       """
 
 
-  Scenario: can redefine entity type's sub
+  Scenario: can redefine entity type's sub, cannot redefine unchanged
     When typeql schema query
       """
-      redefine entity child sub person;
```

Comment:
Kinds are not necessary, removed in some cases so as not to confuse the reader + diverse testing.